qa/workunits/rbd/cli_generic.sh

```diff
@@ -27,6 +27,37 @@ remove_images() {
     done
 }
 
+test_diff() {
+    local pool_name=xrbddiff1
+    local image_name=xtestdiff1
+    local snapshot_name=snap1
+    local image_size=1M
+    ceph osd pool create $pool_name 8
+    rbd create --thick-provision --size $image_size $pool_name/$image_name
```

Comment:
Nit: probably no need to create a new pool since you aren't testing cross-pool functionality